source/systems/gaea_user_guide.rst

```diff
@@ -398,6 +398,61 @@ For example:
 
 Once the job is finished, it should produce an output file.
 
+Displaying Graphics on Compute Nodes
+====================================
+
+.. note::
+
+  Forwarding the display from the compute nodes is an extremely slow process,
+  as three hops are needed to get back to the user's local system. You should
+  only forward from the login nodes.
+
+Follow these steps to display graphics from the compute node.
+
+First terminal
+--------------
+
+Log in to the Gaea system:
+
+.. code-block:: shell
+
+  $ ssh <username>@gaea-rsa.princeton.rdhpcs.noaa.gov
+
+Then request access to a compute node, to start an interactive job:
+
+.. code-block:: shell
+
+  $ salloc -M c5 -t 00:20:00
+
+  salloc: Pending job allocation 134833309
+  salloc: job 134833309 queued and waiting for resources
+  salloc: job 134833309 has been allocated resources
+  salloc: Granted job allocation 134833309
+  salloc: Nodes c5nXXXX are ready for job
+
+Second terminal
+---------------
+
+Log in to the Gaea system and enable X11 forwarding:
+
+.. code-block:: shell
+
+  $ ssh -X <username>@gaea-rsa.princeton.rdhpcs.noaa.gov
```

Comment:
Should we be specifying the princeton bastion here, rather than the load balancer `gaea-rsa.rdhpcs.noaa.gov`?